unlock-protocol-com/blog/treasure-hunt.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Join Spork's ETHDenver Treasure Hunt with Unlock!
+authorName: Christopher Carfi
+publishDate: February 16, 2023
+description: Are you ready to join our brave hero, Spork, on their mission to collect all the prizes at ETHDenver?
+image: /images/blog/treasure-hunt-share.png
+---
+
+![treasure hunt](/images/blog/treasure-hunt-share.png)
+
+Are you ready to join our brave hero, Spork, on their mission to collect all the prizes at ETHDenver? Spork needs your help to accomplish their quest, and your reward will be a plethora of treasures waiting at each of the five stops on the Treasure Hunt! 
+
+You'll help Spork traverse through the five different locations, each of which has its own unique challenge to overcome before being able to collect the prizes. This is no easy feat, but the rewards are worth it!
+
+Along the way, Spork will meet five different frens that will join them on their journey. These frens will provide guidance and support, sharing their knowledge and expertise so that Spork can complete the quests and claim their rewards. Are you up for the challenge? Ready to embark on this epic adventure? Join Spork today and become part of the ETHDenver Treasure Hunt!
+
```

Comment:
```suggestion
Along the way, Spork will meet new frens that will join them on their journey. These frens will provide guidance and support, sharing their knowledge and expertise so that Spork can complete the quests and claim their rewards. Are you up for the challenge? Ready to embark on this epic adventure? Join Spork today and become part of the ETHDenver Treasure Hunt!

```